src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="http://localhost:8080/"
```

Comment:
Don't use localhost for the action attribute. You can use this link for now https://mate-academy-form-lesson.herokuapp.com/create-application